static/js/filter.js

```diff
@@ -25,16 +25,23 @@
 
     filterToggleSearchInput.addEventListener("click", toggleAliasSearchBar, false);
 
-	function filterInputWatcher(e) {
+	function filterInputWatcher(input) {
 
-		const query = e.target.value.toLowerCase();
+        let query = input;
+        
+        if (input.target) {
+            query = input.target.value.toLowerCase();
+        }   
```

Comment:
`let` reasoning: this function was retro fit to run on page load if there was a value in the search filed. (Use case: user refreshes the page while some aliases are being filtered)

Reply:
cc/ @groovecoder